### ISSUE TITLE
Example page issue

### ISSUE THREAD

**Issue by ipoletti** [codepb/jquery-template#70]
There is a issue in the formatter's example in http://codepb.github.io/jquery-template/index.html
```
<div data-content="post" data-template="SameCaseFormatter"
    data-template-format="upper"></div>
```
The attribute is **data-format** instead of data-template

**Comment by codepb** [comment]
Thanks, I will fix it in the next update

**Comment by codepb** [comment]
Fixed in latest commit 81b3be1255532571a4cf323bcb1d1fbe456392fe
